problem_builder/templates/html/mentoring_with_steps.html

```diff
@@ -14,9 +14,9 @@ <h3>{{ title }}</h3>
     {% endfor %}
 
     <div class="submit">
-      <span class="step-overall-checkmark fa icon-2x fa-fw" aria-label=""
+      <button class="step-overall-checkmark fa icon-2x fa-fw" aria-label=""
```

Comment:
@musmanmalik Switching from `span` to `button` changes the look of the element when running on `master`.
![2020-01-14_11-16](https://user-images.githubusercontent.com/27893385/72352649-689e1500-36c1-11ea-8ebd-d96dac873d30.png)
![2020-01-14_11-32](https://user-images.githubusercontent.com/27893385/72352707-83708980-36c1-11ea-8bf0-7bc94b1cdc63.png)
Changing the element to a button is not the best way of making it focusable.
The same functionality can be achieved by adding `tabindex="0"` to the element's properties.
Check more info [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex).

Reply:
@giovannicimolin Thanks for giving your review.

Can you please let me now where are you testing? Since I have have used Xblock-Sdk for testing where i not able to found this use case. 

"tabindex" could not be used here because it will disturb the element navigation because we are appending this module in various locations at various pages. "tabindex" is recommended to use when content not in actual page like popup modals or dialogue box case. 

So here what we do to accommodate the UI like MRQs,MCQs and multi rating question.

I am trying to reproduce this and will fix this UI issue.

Thank you for highlighting.

Reply:
> Can you please let me now where are you testing? Since I have have used Xblock-Sdk for testing where i not able to found this use case.

We always test on edx-platform at the very least. It needs to work there perfectly since that is the primary audience.

>"tabindex" could not be used here because it will disturb the element navigation because we are appending this module in various locations at various pages. "tabindex" is recommended to use when content not in actual page like popup modals or dialogue box case.

A `tabindex` of 0 should not disturb navigation, it is the default value, and only means maintain the current tab order. 

That said a button is probably preferable with the addition of styling overrides.

Reply:
@musmanmalik As I've mentioned before `tabindex="0"` doesn't interfere on the navigation order, just makes the object focusable.

>Can you please let me now where are you testing? Since I have have used Xblock-Sdk for testing where i not able to found this use case.

I'm testing the changes on a Open edX `ironwood` devstack.

Reply:
@giovannicimolin can you please check I have added some style in order to tackle identified issue in this case.

Reply:
@xitij2000  could you please look into this?

Reply:
@musmanmalik I'll review this soon.

Reply:
@musmanmalik I've tested this again, and the button styles are still being applied to the element.
Without any feedback, when selecting the button: :x:
![hover_no_feedback](https://user-images.githubusercontent.com/27893385/72617186-34bc2d00-3917-11ea-862e-85f4ccea6c9c.png)
The screen reader tool reads "button" without any context when hovering over this.

With feedback, focusing the button: :x:
![hover_feedback](https://user-images.githubusercontent.com/27893385/72617214-443b7600-3917-11ea-814e-e5426dc43dfb.png)

With feedback: :heavy_check_mark: 
![no_hover](https://user-images.githubusercontent.com/27893385/72617298-68975280-3917-11ea-80a8-5ceba2f030a3.png)

Either add more styles to fix the issue or switch the element to use `tabindex=0`.

Reply:
@giovannicimolin  Point 1: Resolved

point 2: If we made span focus-able by tabindex  it will behave as button on button focus.
![span having tabindex](https://user-images.githubusercontent.com/43761905/72618486-fe090580-395c-11ea-9504-6114cc15833d.png)
![span with tabindex](https://user-images.githubusercontent.com/43761905/72618494-02352300-395d-11ea-9c33-f35438377653.png)


point 3 : Already good.

Reply:
@musmanmalik Point 2 is to show that the feedback looks like a button on focus:
![image](https://user-images.githubusercontent.com/27893385/72735413-887a8080-3b7a-11ea-865f-c999e63c45f9.png)
It's still looking like a button (see the screenshot above).

Reply:
@giovannicimolin  Thank you

I have fixed above issue. Could you please check this again.